packages/headless/src/api/commerce/common/response.ts

```diff
@@ -7,6 +7,13 @@ import {Pagination} from './pagination';
 import {Product} from './product';
 import {Sort} from './sort';
 
+export interface RecommendationCommerceSuccessResponse {
```

Comment:
Since `responseId`, `products` and `pagination` are common to both success responses defined in this file, we could extract them to a `BaseCommerceSuccessResponse` interface only used in this file, and declare `RecommendationCommerceSuccessResponse` and `CommerceSuccessResponse` from it.